src/util/system.cpp

```diff
@@ -737,8 +776,6 @@ fs::path GetDefaultDataDir()
 }
 
 static fs::path g_blocks_path_cache_net_specific;
-static fs::path pathCached;
-static fs::path pathCachedNetSpecific;
```

Comment:
In commit "Move GetDataDir(fNetSpecific) implementation to ArgsManager." (fac964698fab1b269d4414dc54560b29a3eccd18)

Note (no change suggested): I was initially confused why `pathCached` and `pathCachedNetSpecific` were being removed in this commit without also removing `g_blocks_path_cache_net_specific`. But it's removed in a later commit, and the reason for treating it differently just seems to be that it's needed many fewer places, so the `::GetBlocksDir` wrapper function can be dropped right away, unlike `::GetDataDir`

Reply:
My primary objective was to tackle `::GetDataDir` but then it led me to handle `::GetBlocksDir` too. I did not want to modify those two at the same time.